src/utils/zcl_abapgit_diff.clas.abap

```diff
@@ -158,42 +162,27 @@ CLASS ZCL_ABAPGIT_DIFF IMPLEMENTATION.
 
 
   METHOD get_beacons.
-
     rt_beacons = mt_beacons.
-
   ENDMETHOD.
 
 
   METHOD map_beacons.
 
-    DEFINE _add_regex.
-      CREATE OBJECT lo_regex
-        EXPORTING pattern     = &1
-                  ignore_case = abap_true ##NO_TEXT.
-      APPEND lo_regex TO lt_regex_set.
-    END-OF-DEFINITION.
-
     DATA: lv_beacon_idx  TYPE i,
           lv_offs        TYPE i,
           lv_beacon_str  TYPE string,
           lv_beacon_2lev TYPE string,
           lv_submatch    TYPE string,
-          lo_regex       TYPE REF TO cl_abap_regex,
-          lt_regex_set   TYPE TABLE OF REF TO cl_abap_regex.
+          lo_regex       TYPE REF TO cl_abap_regex.
 
     FIELD-SYMBOLS: <ls_diff> LIKE LINE OF mt_diff.
 
-
-    _add_regex '^\s*(CLASS|FORM|MODULE|REPORT|METHOD)\s'.
-    _add_regex '^\s*START-OF-'.
-    _add_regex '^\s*INITIALIZATION(\s|\.)'.
-
     LOOP AT mt_diff ASSIGNING <ls_diff>.
 
       CLEAR lv_offs.
       <ls_diff>-beacon = lv_beacon_idx.
 
-      LOOP AT lt_regex_set INTO lo_regex. "
+      LOOP AT create_regex_set( ) INTO lo_regex.
```

Comment:
7.4 syntax ?

Reply:
Is it?  Wasn't sure, thought abaplint would catch. Will change!

Reply:
@sbcgua did you test it? in that case its a bug in abaplint, can you help opening an issue for it?

Reply:
Certainly does not work on 731

![image](https://user-images.githubusercontent.com/15635498/73473264-24836380-4395-11ea-94fd-c7f3422cdbe8.png)

will open abaplint issue

Reply:
https://github.com/abaplint/abaplint/issues/768